### DISCUSSION TITLE
out_channels for 3D binary classification with SEnets

### DISCUSSION THREAD

**Discussion by tommydino93** [Project-MONAI/MONAI#3519] [Ideas]
Hi All :)

I am performing 3D classification using densenet. However, I would be curious to try also the [SENets](https://docs.monai.io/en/latest/_modules/monai/networks/nets/senet.html). If I'm not mistaken, currently all the SE nets inherit from SENet which is 2D. Are you planning to extend these networks also for 3D?

Thanks a lot in advance!
Tommaso

**Comment by Nic-Ma** [comment]
Hi @tommydino93 ,

I think all the SENet in MONAI can support 3D data, @yiheng-wang-nv could you please help confirm it?

Thanks in advance.

**Reply by tommydino93** [reply]
Hi @Nic-Ma ,

Actually, I expressed myself wrong. You are right. I can put `spatial_dims=3`.

The issue is more on how to set the `out_channels` argument. I am performing binary classification, so I would like to set `out_channels=2` (as I do for DenseNet), but with SENet I get `Unexpected argument` for `out_channels` . What would be the equivalent argument?

Thanks again and sorry for the inaccuracy

**Reply by Nic-Ma** [reply]
Maybe you can try the `num_classes` arg?

Thanks.

**Reply by tommydino93** [reply]
Yes, it works! Thank you very much for your time :)
